metron-platform/metron-parsing/metron-parsing-storm/src/main/java/org/apache/metron/parsers/bolt/ParserBolt.java

```diff
@@ -326,6 +329,10 @@ protected void handleError(String sensorType, byte[] originalMessage, Tuple tupl
     ErrorUtils.handleError(collector, error);
   }
 
```

Comment:
Why are we hashing here?  This seems like it would be something that would maybe impact performance that we can do another way.  Is there no existing field or type we can use?  Why not just pass the Tuple itself?

Reply:
After thinking about this more, hashing the message is definitely not the right answer.  The most recent commit uses a java UUID instead.  We can explore even more efficient options but this should be an improvement.

Reply:
@merrimanr I think that is a better change, unless we can find a consistent and already existing id to use ( which I don't see either ).
We should just move ahead with this approach